docs/learn/universal-profile/metadata/read-profile-data.md

```diff
@@ -161,33 +151,25 @@ const encodedProfileData = await erc725js.getData();
 
 Find all data keys on the [ERC725Y Inspect](https://erc725-inspect.lukso.tech/data-fetcher) tool or in the [erc725 repo](https://github.com/ERC725Alliance/erc725.js/tree/develop/schemas).
 
-</details>
-
-In the above example, we retrieve encoded `LSP3Profile` and `LSP1UniversalReceiverDelegate` from a Universal Profile.
-
-<details>
-    <summary>Show code snippet</summary>
-
-```js
+```js title="Get data of the LSP3Profile and LSP1UniversalReceiverDelegate data keys"
 import { ERC725, ERC725JSONSchema } from "@erc725/erc725.js";
 import profileSchema from "@erc725/erc725.js/schemas/LSP3ProfileMetadata.json";
 
 const erc725js = new ERC725(
-  profileSchema as ERC725JSONSchema[],
-  "0x03B2689E4843ca56B2A933e7eC1E1ee6C3e6982e",
-  "https://rpc.testnet.lukso.network",
-  {
-  ipfsGateway: "https://api.universalprofile.cloud/ipfs",
-  }
+profileSchema as ERC725JSONSchema[],
```

Comment:
I'm not sure if the formater did it but I think it is more readable with the indent

Reply:
fixed! 🫡